packages/libs/eda/src/lib/core/components/computations/plugins/abundance.tsx

```diff
@@ -165,22 +165,12 @@ export function AbundanceConfiguration(props: ComputationConfigProps) {
     visualizationId
   );
 
-  const collectionVarItems = useMemo(() => {
-    return collections
-      .filter((collectionVar) => {
-        return collectionVar.normalizationMethod
-          ? collectionVar.normalizationMethod !== 'NULL'
-          : true;
-      })
-      .map((collectionVar) => ({
-        value: {
-          collectionId: collectionVar.id,
-          entityId: collectionVar.entityId,
-        },
-        display:
-          collectionVar.entityDisplayName + ' > ' + collectionVar.displayName,
-      }));
-  }, [collections]);
+  const keepCollections =
+    removeAbsoluteAbundanceVariableCollections(collections);
```

Comment:
just so i understand for the future, even if the actual value returned doesnt change this is still an issue bc of the re-rendering?

Reply:
In this case because `keepCollections` an array, its "referential value" changes with every invocation of `removeAbsoluteAbundanceVariableCollections`. React hooks don't check deep equality of their dependencies, just the reference to the array storing the data (this reference isn't available to the developer, unfortunately, unlike in Perl).